packages/jmix-react-ui/src/ui/menu/sub-menu-item/SubMenuItem.tsx

```diff
@@ -1,10 +1,15 @@
 import React from "react";
 import { Menu, SubMenuProps } from "antd";
 
-interface Props extends SubMenuProps {}
+interface Props extends SubMenuProps {
+  caption?: React.ReactNode
+}
 
-export const SubMenuItem: React.FC<Props> = (subMenuItemProps: Props) => {
+export const SubMenuItem: React.FC<Props> = ({caption, ...subMenuItemProps}: Props) => {
   return (
-    <Menu.SubMenu {...subMenuItemProps}/>
+    <Menu.SubMenu 
+      {...subMenuItemProps}
+      title={caption}
```

Comment:
If `caption` is not specified and `title` is specified we will override provided title with undefined (caption).  I would vote for making caption required, otherwise we should implement more clever merging.